specification/synapse/data-plane/Microsoft.Synapse/preview/2022-03-01-preview/runNotebook.json

```diff
@@ -0,0 +1,562 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-03-01-preview",
+    "title": "ArtifactsClient"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "type": "oauth2",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "description": "Azure Active Directory OAuth2 Flow"
+    }
+  },
+  "paths": {
+    "/notebooks/runs/{runId}": {
+      "put": {
+        "tags": [
+          "RunNotebook"
+        ],
+        "operationId": "RunNotebook_CreateRun",
+        "x-ms-examples": {
+          "Run Notebook": {
+            "$ref": "./examples/RunNotebook_Create.json"
+          }
+        },
+        "description": "Run notebook",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/NotebookRunId"
+          },
+          {
+            "name": "RunNotebookRequest",
+            "in": "body",
+            "description": "Run notebook request payload.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RunNotebookRequest"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted.",
+            "headers": {
+              "location": {
+                "description": "URI to poll for asynchronous operation status.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/RunNotebookResponse"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
```

Comment:
Please review https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#Handling-errors for `x-ms-error-code` header.

Reply:
The other existing Synapse APIs don't have such header. Is it mandatory?